services/vision/mlvision/detector.go

```diff
@@ -51,17 +51,35 @@ func attemptToBuildDetector(mlm mlmodel.Service) (objectdetection.Detector, erro
 			return nil, err
 		}
 
-		locations := unpack(outMap, "location")
+		locations, err := unpack(outMap, "location")
+		if err != nil {
+			return nil, err
+		}
 		if len(locations) == 0 {
```

Comment:
if the original "location" is not unpacked, would you ever get to this line?

Reply:
I believe so because they are using different keys to get data out of `outputMap`